app/components/password_toggle_component.html.erb

```diff
@@ -1,9 +1,9 @@
-<%= content_tag(:'lg-password-toggle', class: css_class) do %>
+<%= content_tag(:'lg-password-toggle', **tag_options) do %>
```

Comment:
Previously this component would forward all additional initializer attributes to the `ValidatedFieldComponent`. This made it easier to apply options to the field as an optimized use-case (contrast with new `field_options`), but (a) was inconsistent with other components which would apply additional attributes to the root element and (b) made it difficult or impossible to assign additional attributes to the root element (e.g. extra CSS classes).